### PR TITLE
Feature/esg demo mode

### PR DESCRIPTION
**Pull request by mattcarter**
**TL;DR -**
Merging in the ESG feature branch

**What changed?**

- ESG hooks and CTA buttons

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by leangseu-edx — APPROVED** [review]

**Comment by jansenk** [comment]
> Need to update `package.json` and `package-lock.json` versions

I don't believe we actually need to update package-lock, just package

**Review by nsprenkle — APPROVED** [review]
